resources/views/manager/assessment_plan.html.twig

```diff
@@ -14,7 +14,9 @@
 
     {% include "common/header" with { 'header': { 'title': title } } %}
 
-    {% include "manager/notification" %}
+    {% if hide_demo_notification %}
```

Comment:
Should really rename this to `show_demo_notification` 😛

Reply:
I think I changed it because I thought it might be conflicting with the `show_demo_notification` variable from the notification composer. However, I'm testing now and looks like its working.

Reply:
Nvm, its definitely interfering with the value in DemoNotificationComposer. Ill use `display_demo_notification`